app/assets/stylesheets/refreshments/_invoice.scss

```diff
@@ -24,11 +24,9 @@ $copy-icon-color: #fff;
   border-top: 1px solid $border-color;
 
   &__loading {
-    position: absolute;
-    background-color: $loading-bg;
-    width: 100%;
-    height: 100%;
-    z-index: 1;
+    flex: 1 0 0;
+    margin: auto auto;
```

Comment:
Unexpected longhand value 'auto auto' instead of 'auto'     shorthand-property-no-redundant-values